### DISCUSSION TITLE
Sir may I join team

### DISCUSSION THREAD

**Discussion by hipexscape** [ROM-builders/temporary#19561] [General]
My git username - acex69
My telegram - @raizel_69

My dt
https://github.com/acex69/dt-aosp
https://github.com/acex69/mt6768-common

Successful build (have more )
https://cirrus-ci.com/build/6082062084472832

**Comment by hipexscape** [comment]
Please approve @Apon77 because I am a student and I don't have time to wait for approval. I have money but I can't waste on buying server. I have to study with that

Please sir 
 
I know everything in git that a maintainer should know

**Reply by Apon77** [reply]
When you will be eligible. You will be added. For now, your accounts seems need few more experience!

**Reply by hipexscape** [reply]
Ok
